models/gpt2/export.py

```diff
@@ -0,0 +1,92 @@
+"""Export the GPT-2 model and include all required coefficients."""
+
+import absl.app
+import absl.flags
+import iree.compiler.tools as compiler
+import jax
+import jax.numpy as jnp
+import numpy as np
+import pathlib
+
+# Local classes.
+import config
+import model
+
+from iree.jax import Program, store_global
+from jax.core import ShapedArray
+from os import path
+
+FLAGS = absl.flags.FLAGS
+
+def CreateGpt2Model(name, B, K, S, T):
```

Comment:
Can you document what `B`, `K`, `S`, and `T` are? Maybe like to a page or paper talking about the model architecture.

Reply:
Done along with replicated in config.py